docs/photoniq/fingerprinting/benefits.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 2
+title: Features and Benefits
+---
+
+For businesses with customers worldwide, Fingerprint provides a comprehensive solution that empowers privacy-first identification for enhanced personalization and customization in developing business strategies.  
+Some of these features include: 
+
+### Resilient Fingerprinting​
+
+The Fingerprint service generates fingerprints that persist even after visitors take actions to avoid tracking. These actions include:
+
+- Using incognito/private browsing modes
+- Clearing cookies
+- Upgrading browser version
+
+### Browser-Side Agent
+
+Fingerprint uses a browser-side agent. This agent (a browser-side JavaScript library) automatically loads, collects, and transmits information about a visitor's browser to the Fingerprint service. Fingerprint processes this information using proprietary algorithms and advanced machine-learning models to identify web visitors. Some data types collected include user agents, screen resolutions, CPU counts, and others.
+
+### Server-Side Detection
+
+Fingerprint also uses server-side detection for its geolocation service to determine the visitor's location, including city, state, and zip code. It determines if users access your website via a VPN and compares the visitor's browser time zone with the source address's time zone and a list of well-known public VPN servers to identify the VPN used.
```

Comment:
technically Fingerprinting doesnot have its `geolocation service`. It uses the IP & ASN information though. Dont know if this needs to be rephrased

Reply:
Done